src/manual/design/apis/reference-format.md

```diff
@@ -25,6 +25,15 @@ AMQP exchanges are documented. This is useful for **automatic generation** of:
 **Note**, these formats are not completely stable and may change over time, or
 merge into one.
 
+## Reference Manifest
+
+All services are linked in http://references.taskcluster.net/manifest.json.
```

Comment:
```md
`http://references.taskcluster.net/manifest.json`
```

Also is that still true? @imbstack, or haven't we moved the manifest to the docs site yet?